cmd/pmm-configurator/user.go

```diff
@@ -44,7 +44,7 @@ func createUserHandler(w http.ResponseWriter, req *http.Request) {
 	} else if result != "success" {
```

Comment:
fixed in https://github.com/percona/pmm-manage/pull/9/commits/c0ae429e079e9a55606c3762443fd05f62fdee01